001-IntroToKubernetes/Student/08-storage.md

```diff
@@ -19,11 +19,7 @@ In this challenge we will be creating Azure data disks and using the Kubernetes
 		- Connect to the new pod and you should observe that the “contentdb” database is no longer there since the pod was not configured with persistent storage.
 	- Delete your existing MongoDB deployment
 - Redeploy MongoDB with dynamic persistent storage
-	- **NOTE**: Some types of persistent volumes (such as Azure disks) are associated with a single zone: see [here](https://docs.microsoft.com/en-us/azure/aks/availability-zones#azure-disks-limitations). Since we enabled availability zones in challenge 3, we need to guarantee that the two volumes and the node that the pod runs on are in the same zone.
```

Comment:
Larry modified this section to still call out that disks are located in specific zones.  I prefer his edit of this vs just deleting this section wholesale.